src/data/faq.json

```diff
@@ -1292,13 +1292,13 @@
                                     "<b>Procedure</b>",
-                                    "<ul><li>You can reach the hotline exclusively by telephone.\nCall number: 0800-7540002 (in Germany) or +49.30.49875402 (international)\nPlease check here when you can reach the hotline:\n<a href='#international_phone_numbers' target='_blank'>FAQ: How to reach us</a></li><li>Step 1: Answer the questions of the hotline employee.</li><li>Step 2: You will receive a call back from this employee to give you the TAN. This TAN is valid for one hour.</li><li>Step 3: If the test was already registered, remove the testit with no result from the Corona-Warn-App.</li><li>Step 4: On the Home Screen of the CWA, select the 'Next' button in function 'Are you getting tested?'.\nBelow the question 'Your PCR test was positive?' you will find the function 'Enter TAN for PCR test'. Select this function and enter the 10-digit TAN that was provided to you.</li><li>Your positive test result is now registered in the app. Now you can warn others via the app.</li></ul>",
+                                    "<ul><li>You can reach the hotline exclusively by telephone.\nCall the number: 0800 7540002 (in Germany) or +49 30 498 75402 (international).\nYou can check the operating hours of the hotline under:\n<a href='#international_phone_numbers' target='_blank'>FAQ: How to reach us</a>.</li><li>Step 1: Answer the questions of the hotline employee.</li><li>Step 2: You will receive a call back from this employee to give you the TAN. This TAN is valid for one hour.</li><li>Step 3: If the test was already registered, remove the test with no result from the Corona-Warn-App.</li><li>Step 4: Under the Status tab of the CWA, select the 'Continue' button in the section 'Manage Your Tests'.\nBelow the question 'Your PCR test was positive?' you will find the function 'Enter TAN for PCR Test'. Select this function and enter the 10-digit TAN that was provided to you.</li><li>Your positive test result is now registered in the app. Now you can warn others via the app.</li></ul>",

```

Comment:
Since almost all other FAQ articles do not put "Status" in quotes, when referring to the Status tab, I did not do it either. I think this is acceptable for a single word reference, unlike when referring to a phrase like "Manage Your Tests".

Reply:
Agreed!